src/Components/ProjectTile.js

```diff
@@ -0,0 +1,71 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown, faChevronUp } from '@fortawesome/free-solid-svg-icons';
+import { useHistory } from "react-router-dom";
+import Button from "@mui/material/Button";
+import Service from '../Service';
+import {ReactSession} from "react-client-session";
+
+//
+//       Component: ProductTile
+//       Description: This component wraps the project in its separate box with an upvote and downvote.
+//
+//       Inputs:
+//           - NA
+//       Outputs:
+//          - NA
+const ProjectTile = ({ products, index, setProducts }) => {
+
+  const history = useHistory();
+  const goTo = (product) => () => {
+    history.push(`/${product}/getFeature`);
+  };
+  const capitalizeFirstLetter = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
+
+  const deleteDiv = (id) => {
+    const el = document.getElementById(id);
+    Service.remove('/'+id+'/delete').
+      then(data => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8494981)
```suggestion
      then((data) => {
```